google/cloud/spanner/keys.h

```diff
@@ -37,7 +33,7 @@ template <typename RowType>
 class KeyRange;
 
 namespace internal {
-::google::spanner::v1::KeySet ToProto(KeySet const& keyset);
+::google::spanner::v1::KeySet ToProto(KeySet keyset);
```

Comment:
Just include "internal/keys.h" instead?

Reply:
On general principle, I'd prefer not to include an internal header in a public header if I am able.

Reply:
I would proffer that we should prefer to include the internal header over forward declaring its contents.  We're including it for implementation-detail reasons, so the include is an implementation detail too.  For what it worth, there are already nine internal-header includes in g/c/s/*.h.

Reply:
Forward declaration resolved by removing internal header.